config/jail.conf

```diff
@@ -256,6 +256,9 @@ logpath  = %(auditd_log)s
 
 [apache-auth]
 
+# To also protect web apps with default message patterns,
+# use the following filter :
+#filter  = apache-auth[extendregex="<protect_web_apps>"]
```

Comment:
> impact would be just to have those regexes enabled?

I don't like to enable regex's that not affected most of the people (that also theoretically may be vulnerable)

> Making it optional as in here makes those regexes not tested anyhow so potentially making them not operating correctly or even harmful.

But only as long as it is not covered (see how I made it with sshd-aggressive in #1209, what also merged now, so we can add zzz-protect-web-apps test-case enabled this)